eyepop/compute/api.py

```
@@ -48,36 +48,34 @@ async def fetch_new_compute_session(
-    except aiohttp.ClientResponseError as e:
-        if e.status == 404:
-            need_new_session = True
-        else:
+        except aiohttp.ClientResponseError as e:
             raise ComputeSessionException(
                 f"Failed to fetch existing sessions: {e.message}",
             ) from e
-    except Exception as e:
-        raise ComputeSessionException(f"Unexpected error fetching sessions: {str(e)}") from e
+        except Exception as e:
+            raise ComputeSessionException(f"Unexpected error fetching sessions: {str(e)}") from e
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)